README.md

```diff
@@ -240,4 +240,8 @@ As of February 2021, index resources that don't support tags aren't applicable t
 May 2020: Microsoft.DocumentDB/databaseAccounts/ipRangeFilter updated from a string property to an array.  Please re-author your custom definitions to support the property as an array.  
 July 2020: The alias Microsoft.Sql/servers/securityAlertPolicies/emailAddresses[] and related policies were deprecated. 
 
+### AKS dataplane policies that evaluate containers does not support container exclusions currently.  
```

Comment:
dataplane is the internal term. Docs should refer to what "Resource Provider Modes" are affected  (https://docs.microsoft.com/en-us/azure/governance/policy/concepts/definition-structure#resource-provider-modes)

Reply:
good call, updated!